obspy/clients/fdsn/mass_downloader/__init__.py

```diff
@@ -521,6 +526,15 @@
                      CircularDomain, GlobalDomain)  # NOQA
 
 
+__all__ = [native_str(i) for i in (
+    'MassDownloader', 'Restrictions', 'Domain', 'RectangularDomain',
+    'CircularDomain', 'GlobalDomain')]
+
+msg = ('At least some parts of FDSN Mass downloader might not '
+       'work with old scipy versions <0.12.0 (installed: {})')
+warnings.warn(msg.format(scipy.__version__))
```

Comment:
Should this warning not only be raised if people actually have an old scipy versions?